doc/source/parameters/dem/floating_mesh.rst

```diff
@@ -0,0 +1,51 @@
+Floating Mesh (Solid Objects)
+--------------------------------------
+Floating meshes (solid objects) are finite (limited) auxiliary objects that can be stationary or moving. Rotating impellers, sliding surfaces, and finite stoppers are examples of floating meshes. Note that the main differences between floating meshes and floating walls are: 1. floating wall is a surface, while a floating mesh can be a volume or a plane; 2. floating wall is infinite, while floating mesh is finite; 3. floating wall is stationary while floating mesh may be stationary or moving.
+
+.. note:: 
+    Solid objects (floating meshes) in Lethe have to be defined using triangular (simplex) meshes.
+
+This subsection explains the solid objects (floating meshes) information. First of all, the ``number of solids`` is specified. Then for each solid object, we need a ``mesh`` subsection. In these subsections, the ``type``, ``initial refinement``, and other properties of the objects are defined. Note that ``simplex`` must be ``true`` for all the objects, since the solid objects are defined using simplex meshes.
+
+.. code-block:: text
+
+ subsection floating walls
+  # Total number of floating walls
+    set number of solids  = 1
+      subsection  solid object 0
+        subsection mesh
+            set type                = gmsh
+            set file name           = none
+            set simplex             = true
+            set initial refinement  = 0
+        end
+    
+        subsection translational velocity
+            set Function expression = 0 ; 0 ; 0
+        end
+
+        subsection angular velocity
+        	    set Function expression = 0 ; 0 ; 0
+        end 
+
+        subsection center of rotation
+            # X COR
+            set x = 0
+            # Y COR
+            set y = 0
+            # Z COR
+            set z = 0
+        end
+    end
+ end
+
+* ``number of solids`` parameter defines the total number of floating meshes we wish to use during the simulation.
```

Comment:
```suggestion
* ``number of solids`` parameter defines the total number of floating meshes to be used during the simulation.
```

Reply:
I think the original version by Shahab is better, since it is active voice.

Reply:
Thank you both, but I think I will keep the active voice. :)